dev.env

```diff
@@ -10,6 +10,7 @@ export FLASK_ENV=development
 
 # Setup for Sentry, our error tracking software
 export SENTRY_DSN=
+export SENTRY_ENVIRONMENT=development
```

Comment:
This has been in my `.env` for ages but I realized that I hadn't shared it with everyone.